Source/Scene/SceneTransitioner.js

```diff
@@ -292,7 +292,7 @@ define([
 
     function destroyMorphHandler(transitioner) {
         var tweens = transitioner._currentTweens;
-        for ( var i = 0; i < tweens.length; ++i) {
+        for (var i = 0; i < tweens.length; ++i) {
```

Comment:
Nope, we want this whitespace change so this for statement matches our formatting.  The extra new lines added should be removed though.  Thanks @esraerik!